pages/index.tsx

```diff
@@ -307,7 +307,7 @@ const Home: NextPage = () => {
     href="...&ca=1"
     style="position: absolute; inset: 0;"
   />
-</div>`}
+</div>}
```

Comment:
The line was #294 https://github.com/PRX/Play-Next.js/pull/110/files#diff-aa98fd0757d0e1741503c50cfafb7726939d19819638bbe8e030a27adfec34a3R294

Where there is a:
```
<div>`}
```

that should be:
```
</div>`}
```